### PR TITLE
Slick Slider with "peek" on APro

### PR DESCRIPTION
**Pull request by strikemike2k**

### CODE REVIEW

**Comment by strikemike2k** [comment]
[{APro} Slick sliders; 'peek' method](https://trello.com/c/L1XIXdSO/21-apro-slick-sliders-peek-method)

File: footer.php [hunk 1]
```diff
@@ -9,42 +9,43 @@
...
-<!-- Browser Sync -->
-<!-- Do we need this? *Skylar -->
-<script id="__bs_script__">//<![CDATA[
-  document.write("<script async src='http://HOST:3000/browser-sync/browser-sync-client.js?v=2.26.3'><\/script>".replace("HOST", location.hostname));
-//]]></script>
+      <?php wp_footer(); ?>
+      <!-- Browser Sync -->
+      <!-- Do we need this? *Skylar -->
+      <script id="__bs_script__">//<![CDATA[
+        document.write("<script async src='http://HOST:3000/browser-sync/browser-sync-client.js?v=2.26.3'><\/script>".replace("HOST", location.hostname));
```
Comment (shadoath) [inline footer.php]: I'm not that familiar with the BrowserSync setup. @strikemike2k can we wrap this in a dev_env() check?

File: page-templates/home.php [hunk 2]
```diff
@@ -8,24 +8,34 @@
 <?php get_template_part( 'template-parts/home/slider' ); ?>
 
 <?php
-//TODO Set variables section here
-get_template_part( 'template-parts/home/section' );
-//TODO Set unique variables for next section here
-get_template_part( 'template-parts/home/section' );
-//Repeat twice more ^^
+  foreach(['Culture', 'Travel', 'Gear', 'How To'] as $category){
```
Comment (shadoath) [inline page-templates/home.php]: Oh, yes this is a nice way to load our Categories.
When we have production data we could run a SQL query for them....although that might be superfluous.

File: page-templates/home.php [hunk 3]
```diff
@@ -8,24 +8,34 @@
...
+    include(locate_template('template-parts/home/section.php', false, false));
+  }
 
-//Videos section
-get_template_part( 'template-parts/home/videos' );
+  // Videos section
+  include(locate_template('template-parts/home/videos.php', false, false));
```
Comment (shadoath) [inline page-templates/home.php]: Based on [WP docs](https://developer.wordpress.org/reference/functions/locate_template/) I don't think the two `false, false` calls are needed. If the first one is false the second will have no effect.

File: page-templates/home.php [hunk 4]
```diff
@@ -8,24 +8,34 @@
...
-    cellAlign: 'left',
-    //contain: true
+  $(function() {
+    $('.slider').slick({
+      infinite: false,
+      slidesToShow: 3,
+      responsive: [
+        {
+          breakpoint: 1500, // and down
+          settings: { slidesToShow: 3.25 }
```
Comment (shadoath) [inline page-templates/home.php]: Had no idea that you could put in factional slides.

File: src/assets/js/app.js [hunk 5]
```diff
@@ -7,6 +7,7 @@ import Foundation from 'foundation-sites';
 // If you want to pick and choose which modules to include, comment out the above and uncomment
 // the line below
 //import './lib/foundation-explicit-pieces';
-import './lib/flickity-2.2.0';
+import './lib/slick.min';
```
Comment (shadoath) [inline src/assets/js/app.js]: Flickity gone slickity.

File: src/assets/scss/templates/_home.scss [hunk 6]
```diff
@@ -1,11 +1,25 @@
-.section-image{
-  width: 200px;
-}
-
 .hp-section-title {
   @include underline(black, 2px, 0);
 }
 
+.slick-container {
+  max-width: rem-calc(1800);
```
Comment (shadoath) [inline src/assets/scss/templates/_home.scss]: @BCIMedia/design Would it make sense to set `rem-calc(1800)` to a variable for use across all SCSS sheets?

File: footer.php (hunk 1 above)
Comment (strikemike2k) [reply footer.php]: It is. Currently only in the `footer` branch. https://github.com/BCIMedia/AdventurePro-FoundationPress/commit/43f03491cdad718da62e50929d134d0b57df8c23?w=1#diff-44390cabf006de0249316ec5093ed0c6R97

File: page-templates/home.php (hunk 3 above)
Comment (strikemike2k) [reply page-templates/home.php]: Thanks! I've removed the explicit booleans

File: src/assets/scss/templates/_home.scss (hunk 6 above)
Comment (strikemike2k) [reply src/assets/scss/templates/_home.scss]: Done. https://github.com/BCIMedia/AdventurePro-FoundationPress/pull/3/commits/13d5fae722d532334c1b5181f1812e7c289659c7

File: footer.php (hunk 1 above)
Comment (strikemike2k) [reply footer.php]: These BrowserSync changes are now Merged into `staging`

**Pull request merged**